File: Golang/1012.go
package main

import (
	"fmt"
	"math"
)

func main() {
  var a,b,c float64
  
  fmt.Scan(&a, &b, &c,)
  
  triangulo := (a * c) / 2.0

  pi := 3.14159
  circulo := pi * math.Pow(c, 2)
  trapezio := (a + b) * c / 2.0
  quadrado := math.Pow(b, 2)
  retangulo := a * b

  fmt.Printf("TRIANGULO: %.3f\n",triangulo)
  fmt.Printf("CIRCULO: %.3f\n",circulo)
  fmt.Printf("TRAPEZIO: %.3f\n",trapezio)
  fmt.Printf("QUADRADO: %.3f\n",quadrado)
  fmt.Printf("RETANGULO: %.3f\n",retangulo)
}

File: Elixir/1012.exs
pi = 3.14159

va = IO.gets("") |> String.trim()

vetor = String.split(va, " ") |> Enum.map(&String.trim/1)
a = Enum.at(vetor, 0) |> String.to_float
b = Enum.at(vetor, 1) |> String.to_float
c = Enum.at(vetor, 2) |> String.to_float

# A biblioteca math.pow está elevando o valor ao quadrado

triangulo = (a * c) / 2.0
circulo = pi * (:math.pow(c, 2.0))
trapezio = (a + b) * c / 2.0
quadrado = (:math.pow(b , 2.0))
retangulo = (a * b)

IO.puts "TRIANGULO: #{:erlang.float_to_binary(triangulo, decimals: 3)}"
IO.puts "CIRCULO: #{:erlang.float_to_binary(circulo, decimals: 3)}"
IO.puts "TRAPEZIO: #{:erlang.float_to_binary(trapezio, decimals: 3)}"
IO.puts "QUADRADO: #{:erlang.float_to_binary(quadrado, decimals: 3)}"
IO.puts "RETANGULO: #{:erlang.float_to_binary(retangulo, decimals: 3)}"
